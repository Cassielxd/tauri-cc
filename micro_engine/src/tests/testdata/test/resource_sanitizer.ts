Deno.test("leak", function () {
  Deno.openSync("run/001_hello.js");
  Deno.stdin.close();
});
