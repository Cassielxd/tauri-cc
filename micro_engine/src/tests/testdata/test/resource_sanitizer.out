Check [WILDCARD]/test/resource_sanitizer.ts
running 1 test from ./test/resource_sanitizer.ts
leak ... FAILED ([WILDCARD])

 ERRORS 

leak => ./test/resource_sanitizer.ts:[WILDCARD]
error: Leaking resources:
  - The stdin pipe (rid 0) was opened before the test started, but was closed during the test. Do not close resources in a test that were not created during that test.
  - A file (rid 3) was opened during the test, but not closed during the test. Close the file handle by calling `file.close()`.

 FAILURES 

leak => ./test/resource_sanitizer.ts:[WILDCARD]

FAILED | 0 passed | 1 failed ([WILDCARD])

error: Test failed
